pivotal-tracker/README.md

```diff
@@ -0,0 +1,59 @@
+# Pivotal Tracker Integration #
+
+This example parses commit messages for Pivotal Tracker story IDs and adds the commit message as an activity to the story.
+
+Example comments:
+
+```shell
+  [#148528125] Making a change to this story.
+  I made a change to [#148528125] addressing the functionality.
```

Comment:
These look like one item, rather than a list of two examples. @rachelwhitton, are there existing style guidelines for lists like this?